### PR TITLE
Clarifying

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/7378

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-server/Backup-SPFarm.md [hunk 1]
```diff
@@ -200,6 +200,7 @@ Accept wildcard characters: False
 
 ### -ConfigurationOnly
 If set, the backup will contain configuration settings only (no data) for each of the objects to be backed up.
+In some cases that SharePoint estimates that it does not have sufficient space available, `-Force` parameter needs to be used in conjuntion to this parameter.
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-server/Backup-SPFarm.md:203]:
```suggestion
In some cases that SharePoint estimates that it does not have sufficient space available, `-Force` parameter needs to be used in conjunction with this parameter.
```

**Review by dariomws — COMMENTED** [review]
The issue author wording looks good also. Something like:
"Note that this parameter often miscalculates the backup to be a lot bigger than it actually is, erroneously including content databases in the estimate, so -Force has then to be used in conjunction with this parameter."

Or there is some information you don't want to include?

**Comment by tseward** [comment]
Sounds like this should be filed as a product bug instead, no? @TroyStarr thoughts?

**Comment by yogkumgit** [comment]
@get-itips

**Comment by get-itips** [comment]
> Sounds like this should be filed as a product bug instead, no? @TroyStarr thoughts?

Besides that (agreed) the sentence was already there in another part of the document.

**Comment by TroyStarr** [comment]
Customers are welcome to open a support request through Microsoft Support to request an improvement to how the backup size estimation works.

That said, I think it's fine to provide additional guidance in the cmdlet help as well.  Would this wording be better as a separate paragraph in the description of the -ConfigurationOnly parameter?

SharePoint may overestimate the amount of space required for a configuration-only backup, which may result in SharePoint estimating that it doesn't have sufficient space to store the backup at the specified path. Use the `-Force` parameter in this situation to proceed with the backup if you have sufficient space to store the configuration-only backup.

**Comment by get-itips** [comment]
Thanks @TroyStarr applied your suggestion

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**